src/main/java/org/eclipse/yasson/internal/properties/MessageKeys.java

```diff
@@ -88,6 +88,7 @@
     MISSING_VALUE_PROPERTY_IN_ANNOTATION("missingValuePropertyInAnnotation"),
     NUMBER_INCOMPATIBLE_VALUE_TYPE_ARRAY("numberIncompatibleValueTypeArray"),
     NUMBER_INCOMPATIBLE_VALUE_TYPE_OBJECT("numberIncompatibleValueTypeObject"),
+    INACCESSIBLE_CONSTRUCTOR_PROPERTIES_CREATOR("inaccessibleConstructorPropertiesCreator"),
```

Comment:
it's not necessary to use translated message keys for debug level messages -- only INFO and above need those